extension/main.js

```diff
@@ -16,13 +16,31 @@
 	}
 
 	function update() {
-		gitHubNotifCount(function (count) {
+		gitHubNotifCount(function (count, interval) {
+			var period = 1;
+			if (interval !== GitHubNotify.settings.get('interval')) {
+				GitHubNotify.settings.set('interval', interval);
+				period = Math.ceil(interval / 60);
+				if (period < 1) {
```

Comment:
I would probably add something like this to `GitHubNotify.settings`:

```js
getNumber: function (name) {
    return parseInt(this.get(name), 10);
}
```

and call that to avoid having to `parseInt(…)` every time.